config/answers.sam-installer.yaml

```diff
@@ -3,7 +3,9 @@ certs:
   generate: true
   deploy: true
   group: foreman
-katello: true
+katello:
+  package_names:
+    - katello-sam
```

Comment:
katello-common also requires ruby193-rubygem-katello .. aren't you duplicating that dependency?